hw/ip/spi_device/doc/_index.md

```diff
@@ -71,6 +91,16 @@ signal when the SPI interface is idle.
 
 {{< incGenFromIpDesc "../data/spi_device.hjson" "hwcfg" >}}
 
+The TPM submodule requires a separate input port for CS#.
+The TPM submodule and other SPI Device modes are able to be active together.
+The SB distinguishes between the TPM transactions and the other SPI transactions using separate CS# ports.
+Even though both submodules are able to be active, the SB cannot issue a TPM command and a SPI transaction at the same time due to the shared SPI.
```

Comment:
i think saying "due to the SPI IO lines being shared" might be clearer than saying shared SPI.

Reply:
Revised